### PR TITLE
Resumes

### PR DESCRIPTION
**Pull request by g4s8**
#800 - resumes todo

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#873](https://github.com/zerocracy/farm/issues/873) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#873](https://github.com/zerocracy/farm/issues/873) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/850972997); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

File: src/main/java/com/zerocracy/tk/TkJoin.java [hunk 1]
```diff
@@ -36,6 +36,16 @@
  * @author Kirill (g4s8.public@gmail.com)
  * @version $Id$
  * @since 0.20
+ * @todo #800:30min Requests to /join should add an entry to 'resumes.xml'
```
Comment (carlosmiranda) [inline src/main/java/com/zerocracy/tk/TkJoin.java]: @g4s8 let's add a link to the original comment by @yegor256 , for reference:

https://github.com/zerocracy/farm/issues/800#issuecomment-375551970

**Comment by g4s8** [comment]
@carlosmiranda fixed in a6167aa, please look again.

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by carlosmiranda** [comment]
@yegor256 ping

**Comment by g4s8** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@g4s8 why no code?

**Comment by g4s8** [comment]
@yegor256 because I added code in datum: https://github.com/zerocracy/datum/pull/334
now I knot that I had to submit a ticket in datum instead this todo + datum PR (as in https://github.com/zerocracy/farm/issues/653#issuecomment-376193915), but when I pushed this PR I didn't know it.

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 restart merge please,  `SignatureTest.rendersSignature` test failed

**Comment by yegor256** [comment]
@rultor merge

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thank you!

**Comment by ypshenychka** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**